src/app/screens/Settings.tsx

```diff
@@ -99,7 +99,17 @@ function Settings() {
             </p>
           )}
         </Setting>
-        <Setting title={t("language.title")} subtitle={t("language.subtitle")}>
+        <Setting
+          title={t("language.title")}
+          subtitle={
```

Comment:
Would be great if we could avoid the extra link prop.  
Have a look at this: https://www.createit.com/blog/i18next-react-links-inside-translations/  
This seems more flexible. We might have these kinda situations more often in the future with multiple links.

Reply:
What do you mean the remove extra link prop? I had a look at that article and im not sure what problem this is solving for what we are doing here?